docs/devices/ZYCT-202.md

```diff
@@ -47,4 +46,39 @@ sensor:
 ```
 {% endraw %}
 
+## Home assistant automation
+
+Avoid using a the sensor entity as a trigger from this device, especially when looking to trigger actions based on action_group (The variable identifying the channels).
+This is because the device sends an empty mqtt shortly after the desired payload e.g.
+{% raw %}
+```text
+Nov 08 22:50:35 hass.3ke.uk npm[17965]:   zigbee2mqtt:info 11/8/2019, 10:50:35 PM MQTT publish: topic 'zigbee2mqtt/remote_1', payload '{"linkquality":10,"action":"on","action_group":150}'
+Nov 08 22:50:35 hass.3ke.uk npm[17965]:   zigbee2mqtt:info 11/8/2019, 10:50:35 PM MQTT publish: topic 'zigbee2mqtt/remote_1', payload '{"linkquality":10,"action":""}'
+```
+{% endraw %}
+
+When using a conditional to match the action_group, it appears that the state changes so quickly it makes an unreliable trigger.
 
+Instead use an mqtt trigger for automation
```

Comment:
Using the entity is actually the preferred way to integrate, (https://www.zigbee2mqtt.io/integration/home_assistant.html#via-home-assistant-entity). Are you using a recent version of Home Assistant? Otherwise this is a Home Assistant bug.